src/PKSim.Core/CoreConstants.cs

```diff
@@ -39,7 +39,7 @@ public static class CoreConstants
       public static readonly string DEFAULT_CALCULATION_METHODS_FILE_NAME_FOR_MOBI = "AllCalculationMethods";
 
       public static readonly double[] DEFAULT_STATISTIC_PERCENTILES = {10, 25, 30, 37, 50, 63, 70, 75, 90};
-      public static readonly IEnumerable<int> PRETERM_RANGE = Enumerable.Range(24, 17);
+      public static readonly IEnumerable<int> PretermRange = Enumerable.Range(24, 17);
 
```

Comment:
Had to change this because `PKSimCreatePopulation `in the Matlab-Toolbox crashes otherwise
(and I didn't want crate new Matlab-Toolbox-Core and Matlab-Toolbox version for this hotfix release)

Reply:
WHAT?!? why do this have to do with Matlab?

Reply:
Also this constant is not new right? Why would it crash suddenly? And how on earth did you find out, that the crash was becasue of this constant? Matlab is so bad

Reply:
"The night is dark and full of terrors"
https://github.com/Open-Systems-Pharmacology/Matlab-Toolbox-Core/blob/be5e39f109d4625fef6985a17e395d80655ac0d2/code/auxiliaries/DefaultPopulationSettings.m#L21

Reply:
interesting. Can you add a comment to the code and say that IT SHOULD NOT BE RENAMED ?

Reply:
sure. will do with the next PR